lib/src/runner/configuration.dart

```diff
@@ -395,20 +395,19 @@ class Configuration {
     if (other == Configuration.empty) return this;
 
     var foldTraceOnly = other._foldTraceOnly ?? _foldTraceOnly;
-    if (other._foldTraceOnly != null && _foldTraceExcept != null) {
-      foldTraceOnly = other._foldTraceOnly.difference(_foldTraceExcept);
-    }
-
-    if (other._foldTraceExcept != null && _foldTraceOnly != null) {
-      foldTraceOnly = _foldTraceOnly.difference(other._foldTraceExcept);
-    }
-    if (other._foldTraceOnly != null && _foldTraceOnly != null) {
-      foldTraceOnly = other._foldTraceOnly.intersection(_foldTraceOnly);
-    }
-
     var foldTraceExcept = other._foldTraceExcept ?? _foldTraceExcept;
```

Comment:
At this point, I don't think it makes sense to set these by default since they'll *always* either be overridden or end up as `null`.